src/io.ts

```diff
@@ -98,23 +98,23 @@ export function todoFileNameFor(todoData: TodoData): string {
   return createHash('sha256').update(hashParams).digest('hex').slice(0, 8);
 }
 
-export function writeTodosSync(baseDir: string, lintResults: LintResult[]): string;
+export function writeTodosSync(baseDir: string, lintResults: LintResult[]): TodoBatchCounts;
```

Comment:
Curious if you think it should be returning a tuple or a pojo. I think a POJO would leave us space for other info too (like did we create the lintTodo dir?).

Something like:

```
return {
  added: 99,
  removed: 99,
  // space for other things in the future
};
```

Reply:
I definitely don't feel strongly, we can always iterate on this in future breaking change releases.

Reply:
Ya I thought about using a pojo, but honestly, YAGNI. I suppose it just felt simpler to use this. Let's leave for now and we can change later if we want, since we're the only consumer.